CommonTools/TrackerMap/interface/TrackerMap.h

```diff
@@ -611,7 +611,10 @@ class TrackerMap {
       return lay + 33;
     if (det == 3 && part == 2)
       return lay + 37;
-    return -1;  // This can never happen: det and part are comprised between 1 and 3
+    std::cout
```

Comment:
I'd rather avoid touching this code stable since ~ 20 years at the risk of messing it up (I gather `core` group "edits" do not guarantee extensive tests) for a speedup of standalone code (generally not used in production `cmssw` jobs).